toonz/sources/toonzlib/preferences.cpp

```diff
@@ -519,7 +519,7 @@ void Preferences::definePreferenceItems() {
   define(blanksCount, "blanksCount", QMetaType::Int, 0, 0, 1000);
   define(blankColor, "blankColor", QMetaType::QColor, QColor(Qt::white));
   define(rewindAfterPlayback, "rewindAfterPlayback", QMetaType::Bool, true);
-  define(getShortPlayFrameCount, "shortPlayFrameCount", QMetaType::Int, 8);
+  define(shortPlayFrameCount, "shortPlayFrameCount", QMetaType::Int, 8);
```

Comment:
You need to the minimum and the maximum values (to 5th and 6th arguments) to make this value works.
Without the range, the value will be treated as enum controlled by the combo box.

Reply:
Thanks @shun-iwasawa 
This is my first time dealing with the reworked preferences system.  
I like what you did there, just never had to add anything to it yet.